frontend/src/js/concept-trees/ConceptTreeNodeTextContainer.tsx

```diff
@@ -89,7 +92,12 @@ const ConceptTreeNodeTextContainer: FC<PropsT> = ({
   });
 
   return (
-    <AdditionalInfoHoverable node={node}>
+    <AdditionalInfoHoverable
+      node={node}
+      parent={
+        parentId ? getConceptById(parentId, parentId) ?? undefined : undefined
```

Comment:
I guess this is an indication that we should refactor `getConceptById` to return `undefined` instead of `null`, but no need to do that now, I can follow up with a PR here